types.ts

```diff
@@ -239,10 +239,10 @@ export interface BaseAuthentication {
 
   /**
    * When requiresEndpointUrl is set to true this should be the root domain that all endpoints share.
-   * For example, this value would be "example.com" if specific endpoints looked like {custom-subdomain}.example.com.
+   * For example, this value would be "example.com" if specific endpoints looked like \{custom-subdomain\}.example.com.
    *
    * For packs that make requests to multiple domains (uncommon), this should be the domain within
-   * {@link networkDomains} that this configuration applies to.
+   * {@link networkDomain} that this configuration applies to.
```

Comment:
I think this change is incorrect. This should be a link to the pack definition's `networkDomains`, not the authentication's `networkDomain`

Reply:
Done